kani-compiler/src/kani_middle/transform/check_uninit/uninit_visitor.rs

```diff
@@ -0,0 +1,783 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+//
+//! Visitor that collects all instructions relevant to uninitialized memory access.
+
+use crate::kani_middle::transform::body::{InsertPosition, MutableBody, SourceInstruction};
+use stable_mir::mir::alloc::GlobalAlloc;
+use stable_mir::mir::mono::{Instance, InstanceKind};
+use stable_mir::mir::visit::{Location, PlaceContext};
+use stable_mir::mir::{
+    BasicBlockIdx, CastKind, ConstOperand, LocalDecl, MirVisitor, Mutability,
+    NonDivergingIntrinsic, Operand, Place, PointerCoercion, ProjectionElem, Rvalue, Statement,
+    StatementKind, Terminator, TerminatorKind,
+};
+use stable_mir::ty::{ConstantKind, MirConst, RigidTy, Span, TyKind, UintTy};
+use strum_macros::AsRefStr;
+
+#[derive(AsRefStr, Clone, Debug)]
+pub enum SourceOp {
+    Get { operand: Operand, count: Operand, position: InsertPosition },
```

Comment:
Shouldn't position for `Get` always be Before?

Reply:
I can't think of an example where it wouldn't be Before. I will assume it is always Before for now.